detection-rules/spam_campaign_excessive_display_text_with_keywords.yml

```
@@ -0,0 +1,17 @@
+name: "Spam: Campaign with excessive display-text and keywords found"
+type: "query"
+description: |
+  Detects affiliate marketing spam where any link contains display-text greater than 3000 chars and specific values found. 
+source: |
+  type.inbound
+  and length(body.links) > 0
+  and (
+    any(body.links, length(.display_text) >3000)
+  )
+  and any(
+    body.links, regex.icontains(.display_text, '(\bPassword:)', 'Hi.{0,5}Welcome\b')
+    )
+severity: "low"
+tags:
+  - "Suspicious Link"
+  - "Phishing Attempt"
```

```suggestion
  - "Credential phishing"
```

i think this would be more in-line with current taxonomy (assuming it's accurate), until we roll out something new w/ a clearer definition of what "phishing" is

---

I'm going to pull both tags and just make it say spam. They're affiliate links so they're not actually trying to get creds just clicks.